the-list-be/server.js

```diff
@@ -1,19 +1,46 @@
 import Koa from 'koa';
 import Router from '@koa/router';
+import { MongoClient } from 'mongodb';
+import koaBody from 'koa-body';
+import { ObjectId } from 'mongodb';
 
 const app = new Koa();
 const router = new Router();
 
+const uri = 'mongodb://localhost:27017';
+const client = new MongoClient(uri);
+const db = client.db('database');
+const students = db.collection('students');
+
+router.post('/student', async ctx => {
+  const insertResult = await students.insertOne(ctx.request.body);
+  ctx.body = `Insert documents => ${insertResult}, Request body => ${JSON.stringify(ctx.request.body)}`;
+})
+
+router.get('/students', async ctx => {
+  const findResult = await students.find({}).toArray();
+  ctx.body = findResult;
+})
+
+router.delete('/students_del/:id', async ctx =>{
```

Comment:
И так понятно, что он удаляет, потому что тип метода DELETE 🙂

```suggestion
router.delete('/students/:id', async ctx =>{
```